secrets/org.osbuild.rhsm

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+
+import json
+import os
+import sys
+import re
+
+
+def main():
+    rhsm_secrets = {
+        'ssl_ca_cert': "/etc/rhsm/ca/redhat-uep.pem",
+        'ssl_client_key': "",
+        'ssl_client_cert': ""
+    }
+
+    regexKey = re.compile("^[0-9]*-key.pem")
+    regexCert = re.compile("^[0-9]*.pem")
```

Comment:
I think we can get away with using `glob.glob` here instead of the `listdir` + `regex`.

Reply:
Fixed